GW2EIEvtcParser/EIData/ProfHelpers/Necromancer/ScourgeHelper.cs

```diff
@@ -21,6 +21,7 @@ internal static class ScourgeHelper
             new BuffGainCastFinder(SandstormShroudSkill, DesertShroudBuff).UsingDurationChecker(3500),
             // new EXTBarrierCastFinder(DesertShroud, DesertShroud),
             new EXTBarrierCastFinder(SandCascadeSkill, SandCascadeBarrier),
+            new BuffGainCastFinder(SadisticSearing, SadisticSearing),
```

Comment:
You mean BuffLoss?

Reply:
I'm a little bit conflicted about Sadistic Searing, I feel like people seeing it in the rotation would rather think of the burning proc than the fact that the buff was just gained as simply gaining the buff does not mean much, only that a punishment skill was cast, but we already know that and the buff by itself does nothing either.
Manifest Shade is not an instant skill, perhaps there is a way to reliably check the burning proc by checking that the skill was cast?

Reply:
Only one new custom skillID (for activation, keep application on the normal ID) would have been enough no? Unless you did something else with the buff ID.

Reply:
haven't done anything else with the normal buff id, i'll do that and commit